.github/workflows/test_onnxruntime.yml

```diff
@@ -22,11 +22,9 @@ jobs:
 
     runs-on: ${{ matrix.os }}
     steps:
-    - uses: actions/checkout@v2
-
-    - name: Free disk space
-      if: matrix.os == 'ubuntu-20.04'
-      run: |
+      - name: Free disk space
```

Comment:
should we use an action to handle this instead
https://github.com/marketplace/actions/free-disk-space-ubuntu

Reply:
I do not have a strong opinion on this. Whatever is easier.